docs/en/latest/how-to-build.md

```diff
@@ -65,38 +67,40 @@ sudo yum clean all && yum makecache
 sudo yum install -y --downloadonly --downloaddir=./apisix apisix
 ```
 
-Copy `./apisix` folder to the target host, run the following command to install Apache APISIX.
+Then copy `./apisix` folder to the target host and run the following command to install.
 
 ```shell
 sudo yum install ./apisix/*.rpm
 ```
 
 ### Installation via Docker
 
-Please refer to: [Installing Apache APISIX with Docker](https://hub.docker.com/r/apache/apisix).
+Please refer to [Installing Apache APISIX with Docker](https://hub.docker.com/r/apache/apisix).
 
 ### Installation via Helm Chart
 
-Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/apache/apisix-helm-chart).
+Please refer to [Installing Apache APISIX with Helm Chart](https://github.com/apache/apisix-helm-chart).
 
 ### Installation via Source Release Package
 
+Follow the steps below to install Apache APISIX via the source release package.
+
 1. Create a directory named `apisix-2.12.0`.
 
   ```shell
   APISIX_VERSION='2.12.0'
   mkdir apisix-${APISIX_VERSION}
   ```
 
-2. Download Apache APISIX Release source package.
+2. Download the [Apache APISIX source release package](https://downloads.apache.org/apisix/APISIX_VERSION/apache-apisix-APISIX_VERSION-src.tgz).
```

Comment:
The download link won't work, right? until we use curl/wget and that is already mentioned below, so we can remove the link here. On the safe side we can put the latest link but then it would need to be updated every release. So better to remove it.

```suggestion
2. Download the Apache APISIX source release package.
```